src/cellmanager.js

```diff
@@ -391,8 +391,11 @@ export default class CellManager {
 
     clearSelection() {
         (this._selectedCells || [])
-            .forEach($cell => $cell.classList.remove('dt-cell--highlight'));
-
+            .forEach($cell => {
```

Comment:
Doesn't seem correct. The real problem is this array containing null... why does that happen?

if that root cause is harmless then fix is fine.